api/types/ChannelSlackSettings.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+const ChannelSlackSettings = /* GraphQL */ `
+  enum BotConnectionEventType {
+    threadCreated
+  }
+  
+  type BotConnection {
+    threadCreated: String
+  }
+
+	type ChannelSlackSettings {
+		botConnection: BotConnection
```

Comment:
This name is confusing: `botConnection` is not a connection. (as per the Relay spec and how we use it elsewhere) If you want this to be a connection just like all our other connections (which it should be), please make it conformant by adding edges, nodes and page info:

```GraphQL
type BotSettings {
  threadCreated: String
}

type BotConnectionEdge {
  node: BotSettings!
}

type BotConnection {
  pageInfo: PageInfo!
  edges: [BotConnectionEdge!]
}
```

You don't have to build the actual pagination for now, but at least this sets us up to do it properly in the future.